Gemfile

```diff
@@ -1,23 +1,26 @@
 # -*- mode: enh-ruby -*-
-source 'https://rubygems.org'
 ruby RUBY_VERSION
+source 'https://rubygems.org' do
 
-gem 'chef-vault', '>= 2.6.1'
-gem 'faker'
-gem 'ipaddress'
-gem 'highline'
-gem 'mixlib-shellout'
-gem 'parallel'
-gem 'pry'
-gem 'ruby-progressbar'
-
-gem 'rack', '1.6.4'
-gem 'chef', '12.2.1'
-gem 'buff-ruby_engine', '0.1.0'
-gem 'buff-extensions', '1.0.0'
-gem 'fauxhai', '3.6.0'
+  gem 'chef-vault'
+  gem 'faker'
+  gem 'ipaddress'
+  gem 'highline'
+  gem 'mixlib-shellout'
+  gem 'parallel'
+  gem 'poise'
+  gem 'buff-extensions', '1.0.0'
+  gem 'buff-ruby_engine', '0.1.0'
+  gem 'chef', '12.2.1'
+  gem 'fauxhai', '3.6.0'
+  gem 'json'
+  # nio4r 1.2.1 is the last before requiring Ruby 2.2.2
+  gem 'nio4r', '1.2.1'
```

Comment:
I'm using whatever ChefDK provides; but indeed they are beyond 2.2.2 so this version is finally cleared